src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Color[] colors = Color.values();
+        String randomColor = colors[random.nextInt(colors.length)].name();
```

Comment:
use ColorSupplier instead of code duplication. 
```suggestion
        Color[] colors = Color.values();
        String randomColor = colors[random.nextInt(colors.length)].name();
```